Makefile

```
@@ -21,14 +21,15 @@ all: test
 
 ##@ Build
 .PHONY: build
-build: ## Bazel build
-	bazel build //cmd/cip:cip
+build: ## Go build
+	go build -o cip ${REPO_ROOT}/cmd/cip/main.go
```

```suggestion
	go build -o cip ${REPO_ROOT}/cmd/cip/...
```

Should future-proof this a little if we add more to `cmd/cip` later on.

---

Ack.